.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: python
 python: 3.6
 
+env:
+  global:
+    - ENCRYPTION_LABEL: "b7c548322a41"
```

Comment:
As this is invalid on a fork, I think these should be in Travis environment variables. (also `COMMIT_*`).